src/tests/test_jsproxy.py

```diff
@@ -558,42 +613,63 @@ def test_nested_attribute_access(selenium):
     assert self.Float64Array.BYTES_PER_ELEMENT == 8
 
 
+@run_in_pyodide
 def test_destroy_attribute(selenium):
```

Comment:
This one should also be reverted. The new version of the test doesn't work and doesn't really cover the same behavior.